bootstrap/frontend/src/main/scala/io/udash/bootstrap/carousel/UdashCarousel.scala

```diff
@@ -84,7 +85,7 @@ final class UdashCarousel private(content: ReadableSeqProperty[UdashCarouselSlid
         span(`class` := "sr-only", "Next")
       )
     ).render
-    val jqCarousel = jQ(res).asCarousel()
+    val jqCarousel = jQ(res).asInstanceOf[UdashCarouselJQuery]
```

Comment:
What is the impact of all those inlines? Weren't they already inlined or wouldn't they be with `@inline` hint?

Reply:
It doesn't change JS size, but I think we don't need this boilerplate to hide `.asInstanceOf`.